nilearn/datasets/func.py

```diff
@@ -2056,6 +2064,11 @@ def fetch_development_fmri(n_subjects=None, data_dir=None, resume=True,
     dataset_name = 'development_fmri'
     data_dir = _get_dataset_dir(dataset_name, data_dir=data_dir,
                                 verbose=1)
+    keep_confounds = ['trans_x', 'trans_y', 'trans_z', 'rot_x', 'rot_y',
```

Comment:
Should we add to the examples that these are the included confounds ?

Reply:
renamed

Reply:
might be too much?

Reply:
I'm not tied to it -- thought of it just since this is a little unusual compared to the other datasets. But that might be a more general point about using confounds than we want to make !